src/content/Home/index.js

```diff
@@ -266,16 +266,11 @@ const HomePage = () => {
           </Section>
           <Section backgroundColor='lightorange' p="100px">
             <h2>{t("home.howCovEdWorks")}</h2><br />
-            <Grid container direction="row" spacing={3} justifyContent="center">
-              {howItWorks.map((s) => {
```

Comment:
Please delete the declaration of `howItWorks` in line 112. We're not using it anymore so it'll be a dead variable. 

We can also delete svgs referenced within variable (step1, step2, and step3).